website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -127,12 +127,15 @@ Avoid using critical buttons in alerts. We handle the prominence and importance
 
 #### Links
 
-Use [standalone links](/components/link/standalone) when an action takes the user to a new destination (URL). Follow the standalone link [usage guidelines](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=2522%3A8014&t=s1vlvHztHFnd5T9i-1) to determine what variant “type” to use.
+Use [standalone links](/components/link/standalone) when an action takes the user to a new destination (URL). Follow the standalone link [usage guidelines](https://helios.hashicorp.design/components/link/standalone) to determine what variant “type” to use.
```

Comment:
> Follow the standalone link [usage guidelines](https://helios.hashicorp.design/components/link/standalone) to determine what variant “type” to use.

Is this part still necessary since we're now providing them with an official recommendation?

Reply:
It's not necessary. I'll remove it.